testing/trino-tests/src/test/java/io/trino/tests/TestDistributedFaultTolerantEngineOnlyQueries.java

```diff
@@ -58,4 +59,11 @@ protected QueryRunner createQueryRunner()
         }
         return queryRunner;
     }
+
+    @Override
+    @Test(enabled = false)
+    public void testSelectiveLimit()
+    {
+        // FTE mode does not terminate query when limit is reached
```

Comment:
FYI: @arhimondr @losipiuk . FTE does not honor `LIMIT` in this case

Reply:
Yes, at this point FTE does not support early termination (e.g.: for LIMIT)